lib/jsanalyze.js

```diff
@@ -196,14 +196,16 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 			}
 
 			// copy over polyfill and its dependencies
-			// WARNING: REMEMBER TO UPDATE THIS IF 'babel-polyfill' DEPENDENCIES CHANGE!
-			[ 'babel-polyfill', 'core-js', 'regenerator-runtime' ].forEach((moduleName) => {
-				const moduleSrcDir = path.dirname(require.resolve(path.join(moduleName, 'package.json'))),
-					moduleDstDir = path.join(modulesDir, moduleName);
+			// WARNING: REMEMBER TO UPDATE THIS IF '@babel/polyfill' DEPENDENCIES CHANGE!
+			// Because core-js is under @babel/polyfill, it will get copied. require.resolve didn't work here for it anyways
+			[ '@babel/polyfill', 'regenerator-runtime' ].forEach((moduleName) => {
```

Comment:
Should we move this outside of the `if (opts.transpile) {` and add some logic to include `core-js` to always polyfill?